DesignSafe-CI/portal
# task/DES-2259: Add OpenSees STKO icon

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [DES-2259](https://jira.tacc.utexas.edu/browse/DES-2259)

## Summary of Changes: ##
- Added 'OpenSees-STKO' to appIcons array (array of supported app icons)
- Added CSS selector for above class to also use STKO icon

## Testing Steps: ##
1. Go to https://designsafe.dev/rw/workspace/#!/
2. Open 'My Apps' tab
3. Confirm there is an app bin labeled 'OpenSees-STKO' with the STKO icon (see UI photos)
4. Confirm, in that bin, there are two options in the dropdown called 'should be under OpenSees-STKO tray app SP/MP'.

## UI Photos:
![image](https://user-images.githubusercontent.com/43251554/170142914-b69e9574-ea80-4dbb-9edb-de419e4ca40b.png)


## Notes: ##
